config/logstash.yml

```diff
@@ -50,6 +50,16 @@
 #
 # pipeline.unsafe_shutdown: false
 #
+# Defaults to false. Setting this to true will rescue all exceptions in the
+# pipeline.This means that if there are any errors in filters, or if a
+# conditional raises an exception logstash will drop the event, log an
```

Comment:
suggest: "logstash will do all of the following: drop the erroring event, log an error-level message, and continue processing"